### PR TITLE
CCALC - 4032  'Not sure' in ESC should not show 'still to check' vouchers on 30 free hours page

### PR DESCRIPTION
**Pull request by tyagikumarvineet**

### CODE REVIEW

File: app/uk/gov/hmrc/childcarecalculatorfrontend/controllers/MaxFreeHoursInfoController.scala [hunk 1]
```diff
@@ -44,13 +44,13 @@ class MaxFreeHoursInfoController @Inject()(val appConfig: FrontendAppConfig,
...
         tfc.eligibility(request.userAnswers),
-        getESCEligibility(request.userAnswers),
+        esc.eligibility(request.userAnswers),
         tc.eligibility(request.userAnswers),
         request.userAnswers.taxOrUniversalCredits.getOrElse(""))
       )
   }
 
-  private def getESCEligibility(answers: UserAnswers): Boolean = {
+ /* private def getESCEligibility(answers: UserAnswers): Boolean = {
```
Comment (versatil56) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/controllers/MaxFreeHoursInfoController.scala]: Can we remove this comment?

File: app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala [hunk 2]
```diff
@@ -25,10 +25,13 @@ class EmploymentSupportedChildcare extends Scheme {
 
   override def eligibility(answers: UserAnswers): Eligibility = {
     val No = YesNoUnsureEnum.NO.toString
```
Comment (versatil56) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala]: Let's use the constants we've already defined in childcareconstants

File: app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala [hunk 3]
```diff
@@ -25,10 +25,13 @@ class EmploymentSupportedChildcare extends Scheme {
...
+    val NotSure = YesNoUnsureEnum.NOTSURE.toString
+
 
     val hasParentChildcareCosts: Boolean = answers.childcareCosts.contains(YesNoNotYetEnum.YES.toString)
-    val hasPartnerChildcareVouchers = answers.partnerChildcareVouchers.fold(false)(x => !x.equals(No))
-    val hasParentChildcareVouchers = answers.yourChildcareVouchers.fold(false)(x => !x.equals(No))
+    val hasPartnerChildcareVouchers = answers.partnerChildcareVouchers.fold(false)(x => !(x.equals(No)|| x.equals(NotSure)))
```
Comment (versatil56) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala]: !(x.equals(No)) is inverted logic, more readable to have x.equals(Yes)?

File: app/uk/gov/hmrc/childcarecalculatorfrontend/controllers/MaxFreeHoursInfoController.scala (hunk 1 above)
Comment (tyagikumarvineet) [reply app/uk/gov/hmrc/childcarecalculatorfrontend/controllers/MaxFreeHoursInfoController.scala]: Removed

File: app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala (hunk 2 above)
Comment (tyagikumarvineet) [reply app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala]: Done

File: app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala (hunk 3 above)
Comment (tyagikumarvineet) [reply app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala]: Done

**Pull request merged**